.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: dart
+
+# This is necessary to use proper Chrome. Travis's version of chromium is from
+# 2014.
+sudo: required
+dist: trusty
+addons:
+  chrome: stable
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+dart:
+  - 2.3.2
+  - stable
+  - dev
```

Comment:
need all three of these?

Reply:
Maybe we don't need stable.. but 2.3.2 is the package minimum so it prevents regressions